components/portfolio/components/ProjectDetails.jsx

```diff
@@ -65,16 +65,16 @@ export default function ProjectDetails({ portfolioTitle }) {
                 <i className="fa fa-arrow-left"></i>
                 <span>
                   <span className="link-effect">
-                    <span className="effect-1">Previous Project</span>
-                    <span className="effect-1">Previous Project</span>
+                    <span className="effect-1">Previous</span>
+                    <span className="effect-1">Previous</span>
                   </span>
                 </span>
               </a>
               <a href={nextProject ? `/project-details/${nextProject.title.replace(/\s+/g, '-')}` : ""} className={`nav-btn ${!nextProject ? "disabled" : ""}`}>
                 <span>
                   <span className="link-effect">
-                    <span className="effect-1">Next Project</span>
```

Comment:
Why is this removed?

Reply:
Was changed to fit to smaller screen sizes, else we need to decrease the size of the `Next Project` which can disturb the whole typography of the project